homeassistant/components/media_player/kodi.py

```diff
@@ -482,12 +483,13 @@ def media_content_id(self):
     @property
     def media_content_type(self):
         """Content type of current playing media."""
-        """If item type is unknown to Kodi, return type of first player instead, if any"""
+        """If item type is unknown to Kodi, 
```

Comment:
You'll want to keep a one line summary here, and then include more details later. [Background and an example here](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings). Something like this would work:
```python
    @property
    def media_content_type(self):
        """Content type of current playing media.

        If the media type cannot be detected, the player type is used.
        """
```
Apologies for the nitpicking!

Reply:
oh, of course youre right ))